app/views/admin/social_media_accounts/confirm_destroy.html.erb

```diff
@@ -0,0 +1,21 @@
+<% content_for :context, @social_media_account.service_name %>
+<% content_for :page_title, "Delete social media account" %>
+<% content_for :title, "Delete social media account" %>
+<% content_for :title_margin_bottom, 6 %>
+
+<div class="govuk-grid-row">
+  <section class="govuk-grid-column-two-thirds">
+    <%= form_with url: admin_organisation_social_media_account_path(@socialable, @social_media_account), method: :delete do %>
+      <p class="govuk-body govuk-!-margin-bottom-7">Are you sure you want to delete "<%= @social_media_account.service_name %>"?</p>
+
+      <div class="govuk-button-group">
+        <%= render "govuk_publishing_components/components/button", {
+          text: "Delete",
+          destructive: true,
+        } %>
+
+        <%= link_to("Cancel", admin_organisation_social_media_accounts_path(@social_media_account), class: "govuk-link govuk-link--no-visited-state") %>
```

Comment:
cancel link url is incorrect and does not work. The following looks to work ok:

`<%= link_to("Cancel", admin_organisation_social_media_accounts_path(@socialable), class: "govuk-link govuk-link--no-visited-state") %>`